app/javascript/controllers/funnel_chart_controller.js

```diff
@@ -26,6 +29,7 @@ export default class extends Controller {
       type: "bar",
       data,
       options: {
+        events: [],
```

Comment:
What does this do?

Reply:
it disables hover and click functionality (because value is already shown)